3.8/Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM python:3.8-slim
+FROM python:3.8-slim-buster
 
 RUN apt-get update && \
     apt-get install --no-install-recommends -y \
-        wget=1.20.1-1.1 \
-        zip=3.0-11+b1 \
-        unzip=6.0-23+deb10u2 \
-        git=1:2.20.1-2+deb10u3 \
-        nodejs=10.23.1~dfsg-1~deb10u1 \
-        npm=5.8.0+ds6-4+deb10u2 && \
+        wget \
+        zip \
+        unzip \
+        git \
+        nodejs \
+        npm && \
```

Comment:
Try to don't use the latest, always ping the version.

Reply:
Fixed